src/main/java/stelnet/ui/Renderable.java

```diff
@@ -28,4 +49,29 @@ public void log() {
     public void log(Position position) {
         LogHelper.debug(String.format("Rendered %s in %s", this, position));
     }
+
+    public Size getSize() {
+        if (size == null) {
+            LogHelper.warn("Null size found");
+        }
+        return size;
+    }
+
+    public Location getLocation() {
+        return location;
+    }
+
+    public Position getOffset() {
+        return offset;
+    }
+
+    public void setLocation(Location location) {
+        this.location = location;
+    }
+
+    public void setOffset(Position offset) {
+        this.offset = offset;
+    }
```

Comment:
You could add here class level `@Getter` and keep custom implementation of getSize